Misc/NEWS.d/next/Library/2018-07-29-15-25-15.bpo-34246.xiKq-Q.rst

```diff
@@ -0,0 +1,2 @@
+Use an empty tuple as the default argument for mail_options and avoid it from
```

Comment:
The argument for what function? Using an empty tuple is an implementation detail, please describe the user visible behavior change. For example "```:meth:`smtplib.SMTP.send_message` no longer modifies the content of the *mail_options* argument.```" or something like.

Add also "Patch by Pablo S. Blum de Aguiar."